src/org/parosproxy/paros/extension/option/OptionsJvmPanel.java

```diff
@@ -93,8 +109,45 @@ private ZapTextField getJvmOptionsField() {
 		return jvmOptionsField;
 	}
 	
+	private JLabel getCurrentMemoryLabel() {
+		if (currentMemoryLabel == null) {
+			currentMemoryLabel = new JLabel();
+			updateMemoryLabel(currentMemoryLabel, Constant.messages.getString("jvm.options.memory.current",
+					sbsv.getString(Runtime.getRuntime().totalMemory())));
+		}
+		return currentMemoryLabel;
+	}
+	
+	private JLabel getFreeMemoryLabel() {
+		if (freeMemoryLabel == null) {
+			freeMemoryLabel = new JLabel();
+			updateMemoryLabel(freeMemoryLabel, Constant.messages.getString("jvm.options.memory.free",
+					sbsv.getString(Runtime.getRuntime().freeMemory())));
+			}
+		return freeMemoryLabel;
+	}
+	
+	private JLabel getMaxMemoryLabel() {
+		if (maxMemoryLabel == null) {
+			maxMemoryLabel = new JLabel();
+			updateMemoryLabel(maxMemoryLabel, Constant.messages.getString("jvm.options.memory.max",
+					sbsv.getString(Runtime.getRuntime().maxMemory())));
+		}
+		return maxMemoryLabel;
+	}
+	
+	private void updateMemoryLabel(JLabel labelToUpdate, String text) {

```

Comment:
Instead of text it could require the key and the value, so in this method it would do:
`labelToUpdate.setText(Constant.messages.getString(key, sbsv.getString(value)))`